Restructuring container docs (#886)
CHTC/chtc-website-source: _uw-research-computing/conda-installation.md

Reviewer: Honestly this page is super long... may need a trim in another PR later

## Quickstart: Conda

### Option A - Container (recommended)

Build a container with Conda packages installed inside. Start by reviewing 
the container build instructions for either Apptainer or Docker. Then, when 
following the build instructions, use the example recipes for either a 
`.def` or `Dockerfile`, respectively. 

1. How to build your own container: [Apptainer](apptainer-htc#use-an-apptainer-container-in-htc-jobs) / [Docker](docker-build)
2. Tips and tricks, container recipes: [Example container recipes for Conda](https://github.com/CHTC/recipes/tree/main/software/Conda)
3. Use your container in your HTC jobs: [Apptainer](software-overview-htc.html#use-an-existing-container) / [Docker](docker-jobs.html#use-a-docker-container-in-a-job)

<!-- replace this eventually with conda walkthrough --> 

### Option B - "Packed" Conda Env

Create your own portable copy of your Conda packages:

> > This approach may be sensitive to the operating system of the execution point. 

Reviewer: ```suggestion
> This approach is sensitive to the operating system of the execution point. 
```
